features/FEATURE_BLE/targets/TARGET_CORDIO/source/CordioBLE.cpp

```diff
@@ -510,7 +510,7 @@ void BLE::stack_setup()
 
     stack_handler_id = WsfOsSetNextHandler(&BLE::stack_handler);
 
-    HciSetMaxRxAclLen(100);
+    HciSetMaxRxAclLen(MBED_CONF_CORDIO_DESIRED_ATT_MTU);
```

Comment:
Clearly this was an oversight, there isn't a config for acl on host side. We should really create one, otherwise if you want o use att_mtu then you just need to add the header
```suggestion
    HciSetMaxRxAclLen(MBED_CONF_CORDIO_DESIRED_ATT_MTU + L2C_HDR_LEN);
```